### DISCUSSION TITLE
请问绑定完了之后又要怎么操作呢？

### DISCUSSION THREAD

**Discussion by 1w5** [walinejs/waline#1364] [Q&A] [answered]
https://forum.leancloud.cn/t/2022-8/25408 需要 LeanCloud 上绑下域名用 serverURL 配一下

_Originally posted by @lizheming in https://github.com/walinejs/waline/issues/1338#issuecomment-1232765106_

**Comment by lizheming** [comment, marked as answer]
配置 LEAN_SERVER 环境变量为 LeanCloud 绑定的域名，重新部署应用
